src/components/picture/_children/source.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from "prop-types";
+import formatSrcset from "../../../utils/format-image-resizer-src";
```

Comment:
Should rename `formatSrcset` to `formatSrc` here for consistency.

Reply:
Thanks! Updated!